packages/config/config/devices/0x0330/zv2833pac.json

```diff
@@ -0,0 +1,211 @@
+{
+	"manufacturer": "Sunricher",
+	"manufacturerId": "0x0330",
+	"label": "ZV2833PAC",
+	"description": "Push-Button Coupler",
+	"devices": [
+		{
+			"productType": "0x0400",
+			"productId": "0xd105"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Launch 1",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Launch 2",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Launch 3",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Launch 4",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "2",
+			"label": "External Switch Type",
+			"description": "External Switch Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Push button",
+					"value": 0
+				},
+				{
+					"label": "Normal On/Off switch",
+					"value": 1
+				},
+				{
+					"label": "3-Way switch",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "Duration of the Multilevel Start Level Change Up/Down",
+			"description": "Set the duration of the Multilevel Start Level Change Up/Down",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 8
+		},
+		{
+			"#": "4",
+			"label": "Associations in Z-Wave network Security Mode",
+			"description": "Associations in Z-Wave network Security Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 15,
+			"defaultValue": 15
+		},
+		{
+			"#": "5",
+			"label": "Scenes sent to the controller for button 1",
+			"description": "Scenes sent to the controller for button 1",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 15,
+			"defaultValue": 15,
+			"options": [
+				{
+					"label": "button short pressed once",
+					"value": 1
+				},
+				{
+					"label": "buton short pressed twice",
+					"value": 2
+				},
+				{
+					"label": "button pressed and held",
+					"value": 4
+				},
+				{
+					"label": "button released",
+					"value": 8
+				}
+			]
```

Comment:
This looks like they are meant to be combined, so this needs one partial parameter per scene (Pressed 1x, Pressed 2x, Press and hold, Release).
I'm sure there is a similar device already that this definition can more or less be copied from.

Same goes for params 6-8, but they can most likely be `$import`-ed from param 5 -> https://zwave-js.github.io/node-zwave-js/#/config-files/using-templates?id=example-3

Reply:
ditto from comment above